src/main/java/com/eppo/sdk/constants/Constants.java

```diff
@@ -32,9 +32,13 @@ public class Constants {
      */
     public static final String RAC_ENDPOINT = "/randomized_assignment/v3/config";
 
+    public static final String BANDIT_ENDPOINT = "/randomized_assignment/v3/bandits";
```

Comment:
Cool if you decide to go down this route please note we'll have basically a separate Fastly behavior in terraform, off-boarding firewall rules, etc.

https://github.com/Eppo-exp/eppo/blob/main/terraform/fastly.tf#L33-L38

What do you think are pros/cons to putting bandits within the config url, like this:

`/randomized_assignment/v3/config/bandits`

Reply:
Thanks for the heads up about terraform! 

No strong opinions on `/randomized_assignment/v3/config/bandits`--do you think that's better because it's related to the config?